### ISSUE TITLE
Remove compile-scope dependency to slf4j-simple

### ISSUE THREAD

**Issue by GoogleCodeExporter** [vadbch/orika#35]
```
slf4j-simple is one implementation of slf4j-api.

As a library orika should log to slf4j (the simple logging facade for java), 
which adds a compile-scope dependency to slf4j-api.

However, it should be up to the user which implementation to use (such as, in 
my case, logback).
```

Original issue reported on code.google.com by `dale.wij...@gmail.com` on 24 Jul 2012 at 7:51

**Comment by GoogleCodeExporter** [comment]
```
Resolved in 1.2.0 branch
```

Original comment by `matt.deb...@gmail.com` on 24 Jul 2012 at 8:34
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Excellent. Thank you Matt.

As a side note, the latest stable/bugfixed version of slf4j is 1.6.6 and for 
logback 1.0.6, but I'm not sure if you follow a specific policy for dependency 
version updates.
```

Original comment by `dale.wij...@gmail.com` on 24 Jul 2012 at 9:08
